schema/2.0.0/corpora_schema.md

```diff
@@ -143,15 +143,18 @@ The following ontology dependencies are *pinned* for this version of the schema.
 
 ### Required Gene Annotations
 
-cellxgene requires ENSEMBL gene identifiers to ensure that all datasets it stores measure the same features and can therefore be integrated.
+cellxgene requires ENSEMBL identifiers for genes and External RNA Controls Consortium (ERCC) identifiers for [RNA Spike-In Control Mixes] to ensure  that all datasets it stores measure the same features and can therefore be integrated.
 
-The following gene annotation dependencies are *pinned* for this version of the schema.
+The following gene annotation dependencies are *pinned* for this version of the schema. For multi-organism experiments, cells from any organism are allowed as long as orthologs from the following organism annotations are used.
 
-| Organism | Required version | Download |
+| Source | Required version | Download |
 |:--|:--|:--|
 | [ENSEMBL (Human)] | Human reference GRCh38 (GENCODE v32/Ensembl 98)] matching [cellranger 2020-A (July 7, 2020) release] | [gencode.v32.primary_assembly.annotation.gtf] |
 | [ENSEMBL (Mouse)] | Mouse reference mm10 (GENCODE vM23/Ensembl 98) matching [cellranger 2020-A (July 7, 2020) release]| [gencode.vM23.primary_assembly.annotation.gtf] |
-|||
+| [ENSEMBL (COVID-19)] | SARS-CoV-2 reference (ENSEMBL assembly: ASM985889v3) | [Sars\_cov\_2.ASM985889v3.101.gtf] |
+| [ThermoFisher ERCC Spike-Ins] | ThermoFisher ERCC RNA Spike-In Control Mixes (Cat # 4456740, 4456739) | [cms_095047.txt] |
```

Comment:
Is there a reason that we are not referencing/downloading the GTF available on this page - https://www.thermofisher.com/order/catalog/product/4456739#/4456739 - ERCC92.fa & ERCC92.gtf sequence and annotation files (.zip)

or using cellranger references to be consistent with the other annotations?

https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/advanced/references

Cell Ranger provides pre-built human (hg19, GRCh38), mouse (mm10), and ercc92 reference packages for read alignment and gene expression quantification in cellranger count.

https://support.10xgenomics.com/single-cell-gene-expression/software/release-notes/build

ERCC reference dataset, ercc92
```
cellranger mkref --genome=ercc92 \
                 --fasta=ercc92.fa \
                 --genes=ercc92.gtf \
                 --ref-version=1.2.0
```

Reply:
This requires to install cellranger, I couldn't find a download link. 

---

The GTF has so much more information that's not needed. The txt file requires no parsing, so it's the easiest to maintain. Unless there's something programmatically advantageous about  the GTF, there's no need to use it and we should stick to the simpler txt file. Having the GTF file means that we need to process it slightly differently than the genes' GTFs because we only want IDs, and parsing the GTF would mean the creation of a file very similar to the text file